### ISSUE TITLE
After update organizer venue add-on organizer image replace with event banner.

### ISSUE THREAD

**Issue by krinaydhanani** [wpeventmanager/wp-event-manager#307]
![image](https://user-images.githubusercontent.com/35419531/89788636-17fa7780-db3d-11ea-91a8-65320db135b5.png)

**Comment by wpem-kishan** [comment]
we have already implemented in new version first image of banner image set as a featured image.

**Comment by krinaydhanani** [comment]
It's not working fine. I got same issue second time.


![image](https://user-images.githubusercontent.com/35419531/90108485-01843400-dd68-11ea-925e-8eedeabeb334.png)

**Comment by ritakikani** [comment]
I also face this same issue.

First i had installed old plugin version, then add some events then update with new version with organizer feature.
then again add events, then i view the event listing page and found that for old events which is entered during old version that all are showing with organizer banner rather than event banner, where as in upcoming widget all banner seen correct, also working fine in single event page.
You can see in screenshot below.


![FireShot Capture 199 - Events – Latest WordPress - localhost](https://user-images.githubusercontent.com/25837982/90306900-222ac600-deef-11ea-815b-4f6e28ce7e05.png)

**Comment by hiteshmakvana** [comment]
Yes, please upgrade with latest version (migration data base process).
